lib/ReCaptcha/ReCaptcha.php

```diff
@@ -70,7 +70,7 @@ public function __construct($secret, RequestMethod $requestMethod = null)
         if (!is_null($requestMethod)) {
             $this->requestMethod = $requestMethod;
         } else {
-            $this->requestMethod = new RequestMethod\Post();
+            $this->requestMethod = new RequestMethod\WP();
```

Comment:
Let's not modify the original library files, perhaps we can pass in the `$requestMethod` in the `ReCaptcha` constructor?